tests/suite/scale_test.go

```diff
@@ -562,21 +571,37 @@ The logs are attached only if there are errors.
 			},
 		)
 	})
-
-	It(fmt.Sprintf("scales upstream servers to %d", upstreamServerCount), func() {
-		const testName = "TestScale_UpstreamServers"
-
-		testResultsDir := filepath.Join(resultsDir, testName)
-		Expect(os.MkdirAll(testResultsDir, 0o755)).To(Succeed())
-
-		runTestWithMetricsAndLogs(
-			testName,
-			testResultsDir,
-			func() {
-				runScaleUpstreams()
-			},
-		)
-	})
+	if upstreamServerCount == ossUpstreamServerCount {
```

Comment:
Can't we just use the `upstreamServerCount`?

Reply:
When I tried that I ran into issues where since upstreamServerCount is dynamically set, the ginkgo blocks only seem to work with static variables, so I would get 0, since it would process all the blocks before actually running the `BeforeAll` where we set the variable.

Reply:
Oh, interesting. I don't think it's worth having this if statement to include the number of upstream servers in the test name. Let's just include both values in the test description: "scales upstream servers to %d for OSS and %d for plus"